src/plugins/dotplot/src/DotPlotWidget.cpp

```diff
@@ -777,26 +782,32 @@ bool DotPlotWidget::sl_showSettingsDialog(bool disableLoad) {
 }
 
 // ask user if he wants to save dotplot first
-void DotPlotWidget::sl_showDeleteDialog() {
-    int answer = QMessageBox::information(this, tr("Save dot-plot"), tr("Save dot-plot data before closing?"), QMessageBox::Yes, QMessageBox::No, QMessageBox::Cancel);
-    bool saveDotPlot;
-
+void DotPlotWidget::sl_showDeleteDialog(bool isCancelable) {
+    int answer = 0;
```

Comment:
Set  `QMessageBox::StandardButton` as type

Reply:
Only integer is acceptable option.
`static int information(QWidget *parent, const QString &title,
                           const QString& text,
                           int button0, int button1 = 0, int button2 = 0);`